ledger/ledger.go

```diff
@@ -0,0 +1,70 @@
+// Copyright 2019 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package ledger
```

Comment:
Twould be nice to have a package-level comment providing some context for what this package is for.

Reply:
Note that packages in this repo are intended to be general in nature, to be used by multiple components. If this package ends up as an implementation detail of Pilot, then its code should be within the Pilot source tree.

Reply:
will add comment.  package will eventually be used in Galley as well.